torch_xla/csrc/tensor_util.cpp

```diff
@@ -796,13 +796,18 @@ std::vector<xla::Literal> ReleaseGilAndTransferData(
 
 std::vector<at::Tensor> XlaDataToTensors(
     absl::Span<const torch::lazy::BackendDataPtr> xla_data,
-    at::ScalarType dest_element_type) {
+    absl::Span<const at::ScalarType> dest_element_type) {
```

Comment:
What's the reason for this change?

Reply:
it seems like we never really call `XlaDataToTensors ` with different `dest_element_type`. @jonb377 are you introducing a new use case? if not we can keep it as a singleton?

Reply:
The reason to make the dest_elem_types a vector is actually the next change - I'm batching the local shard transfers for many tensors into a single XlaDataToTensors call. I probably should have kept this refactor with the upcoming change... But it makes that PR slightly smaller.